templates/lnurlp/index.html

```diff
@@ -56,6 +57,18 @@ <h5 class="text-subtitle1 q-my-none">Pay links</h5>
                   :color="($q.dark.isActive) ? 'grey-7' : 'grey-5'"
                   @click="openQrCodeDialog(props.row.id)"
                 ></q-btn>
+                <q-btn
+                  unelevated
+                  dense
+                  size="xs"
+                  icon="checkmark"
+                  :color="($q.dark.isActive) ? 'grey-7' : 'grey-5'"
+                  type="a"
+                  :href="'https://sendsats.to/' + props.row.username +'@' + domain"
+                  target="_blank"
+                >
+                </q-btn>
+                <q-tooltip>Check LN Address works via external call</q-tooltip>
```

Comment:
I think this is unnecessary and could potentially be a privacy issues. Can you get rid of this button?